### DISCUSSION TITLE
Capture snapshot of OpenGL window with very large resolution

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#333] [Feature Requests]
A way to get a super high res image out of cables. My initial thought was to output a set of images that could be stitched together in photoshop or something. 10,000x10,000px images would make it easy. (would we need an overlap?)

I think it’s a good thing to have as so many people I speak to not only want a site, visual, or something interactive but also a print asset, poster, magazine image, and in some cases a billboard.

If it could work with a perspective camera that would be ace. 
What do you think?

[https://stackoverflow.com/questions/6490728/capture-snapshot-of-opengl-window-with-very-large-image-resolution](url)

**Comment by marioepsley** [comment]
bump :)

**Comment by kikohs** [comment]
Unfortunately, you can't go over a 4096x4096 texture without tilling.

**Reply by marioepsley** [reply]
I can save out 16000x16000px squares at the moment :)

**Reply by kikohs** [reply]
how? My viewport doesn't go above 4k.

**Comment by pandrr** [comment]
depends on gpu and webgl implementation
